roles/kube-gateway/files/nginx.yaml

```diff
@@ -0,0 +1,10 @@
+init_config:
+
+instances:
+
+    -   nginx_status_url: https://localhost:7331/nginx_status/
+        # NOTE(jk0): If enabled, SSL verification will fail when a host is set
+        # to use a planet-labs.com cert, and the ELB uses planet.com.
+        ssl_validation: False
+        tags:
+            -   host:nginx
```

Comment:
I'm not sure if this will override the `host` or `hostname` tags automatically set by dd-agent. I'd prefer to just remove this `tags` block altogether.